addons/website_blog/models/website_blog.py

```diff
@@ -175,8 +175,8 @@ def _default_content(self):
     blog_id = fields.Many2one('blog.blog', 'Blog', required=True, ondelete='cascade', default=lambda self: self.env['blog.blog'].search([], limit=1))
     tag_ids = fields.Many2many('blog.tag', string='Tags')
     content = fields.Html('Content', default=_default_content, translate=html_translate, sanitize=False)
-    teaser = fields.Text('Teaser', compute='_compute_teaser', inverse='_set_teaser')
-    teaser_manual = fields.Text(string='Teaser Content')
+    teaser = fields.Text('Teaser', compute='_compute_teaser', inverse='_set_teaser', translate=True)
```

Comment:
```python
teaser = fields.Text('Teaser', compute='_compute_teaser', inverse='_set_teaser')
``` 

As discussed on discord, if `translate=True` is necessary for UI, we need a comment.